cymetric/cyclus.pyx

```diff
@@ -46,6 +47,24 @@ cdef class _Datum:
     #        free(self.ptx)
 
     def add_val(self, const char* field, value, shape=None, dbtype=cpp_typesystem.BLOB):
+        """Adds Datum values for type conversion.
+
+        Parameters
+        ----------
+        field : pointer to char/str
+            The column name.
+        value : any data type
```

Comment:
`object` is the correct Python terminology for any data type, though this is not incorrect.